ci-operator/config/openshift/openshift-controller-manager/openshift-openshift-controller-manager-master.yaml

```diff
@@ -15,7 +15,7 @@ images:
   inputs:
     bin:
       as:
-      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.13
```

Comment:
I wonder if the build change has to merge first before we can do this.

Let's see how the rehearsal jobs go, then circle back.

Reply:
the rehearsal jobs finished ... not sure they covered this permutation